resources/layout/_footer.php

```diff
@@ -1,13 +1,37 @@
 <div class="columns is-mobile">
     <div class="column has-text-left has-text-light">
-        Left
+        <i class="fas fa-copyright fa-inverse is-hidden-mobile"></i>
+        <a class="is-hidden-mobile" href="https://www.yiiframework.com/" target="_blank" rel="noopener">
```

Comment:
Why this trick is needed?

Reply:
`target="_blank" rel="noopener"`? Security concerns: https://web.dev/external-anchors-use-rel-noopener/